src/librustc_parse/parser/module.rs

```diff
@@ -1,7 +1,7 @@
 use super::item::ItemInfo;
```

Comment:
The whole `parser/module.rs` needs to be merged into `parser/item.rs` since modules are no longer special and are parsed just like other items now.

Reply:
Done in a next commit.